static/js/trees.js

```diff
@@ -0,0 +1,575 @@
+                                            {
+                                                pre:{
+                                                    class: `code-${integration}`,
+                                                    content: links["ARM"]
+                                                }
+                                            }
+                                        ]
+                                    }
+                                },
+                                {
+                                    p:{
+                                        content: "You can find more information: ",
+                                        children: [
+                                            {
+                                                a: {
+                                                    content: "here",
+                                                    href: "https://kairos.io/docs/reference/image_matrix/#image-flavors"
+                                                }
+                                            }
+                                        ]
+                                    }
+                                },
+                                {
+                                    div:{
+                                        class: "modal-buttons",
+                                        children: [
+                                            {
+                                                button: {
+                                                    class: `copy-to-clipboard-${integration}`,
+                                                    content: "Copy Command",
+                                                }
+                                            },
+                                            {
+                                                button: {
+                                                    class: `close-dialog-${integration}`,
+                                                    content: "Close",
+                                                }
+                                            }
+                                        ]
+                                    }
+                                }
+                            ]
+                        }
+                    }
+                    ]
+                }
+            }
+        ]
+    }
+}))
+export const DOWNLOAD_BLADE_TREE = {
+    div: {
+        class: "download-card-wrap",
+        children: [
+            {
+                div: {
+                    class: "download-cards",
+                    children: [
+                        {
+                            div: {
+                                content: "Downloads"
+                            }
+                        }, ...DOWNLOAD_CARDS_TREE
+                    ]
+            }
+            },
+            {
+                a: {
+                    href: "https://github.com/kairos-io/kairos",
```

Comment:
let's point this to the image matrix https://kairos.io/docs/reference/image_matrix/#image-flavors